### PR TITLE
@alloy => [Publishing] Kill default exports in publishing components

### PR DESCRIPTION
**Pull request by kanaabe**
There's something strange going on with Prettier on styled components. It's adding commas to places where we use template literals...still haven't looked into it but it all looks good on Storybooks and tests.

Looks like ^ was happening with my other machine but not my home one so it must be a local issue.

### CODE REVIEW

File: src/Components/Publishing/Byline/AuthorDate.tsx [hunk 1]
```diff
@@ -82,11 +94,10 @@ const StyledAuthor = Text.extend`
...
   }
-  ${props => adjustForCondensed(props.layout)}
-  ${pMedia.sm`
+  ${props => adjustForCondensed(props.layout)} ${pMedia.sm`
     &:before {
       min-width: 8px;
       min-height: 8px;
     }
-  `}
+  `};
```
Comment (kanaabe) [inline src/Components/Publishing/Byline/AuthorDate.tsx]: It's a lot of odd behavior like this ^

**Comment by alloy** [comment]
Woah, epic! :clap:

**Comment by orta** [comment]
very cool 👍

**Comment by alloy** [comment]
Oh no, looks like I never submitted my review comment... Will do so when I get back to my laptop.

File: src/Components/Publishing/index.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ import {
   StandardArticle,
 } from "./Fixtures/Articles"
 
-export default {
+export const components = {
```
Comment (alloy) [inline src/Components/Publishing/index.tsx]: Do you really want this to be named `components`? You’d have to use it like so:

```js
import { components } from "@artsy/reaction/dist/Components/Publishing"
const { Article } = components
```

You can use this ES6 syntax to just export the individual items:

```js
export { Article } from "./Article"
```

And then the import looks like:

```js
import { Article } from "@artsy/reaction/dist/Components/Publishing"
```

**Comment by alloy** [comment]
The PR looks great, feel free to address/ignore my comment about the exports from the index file and merge 👍

File: src/Components/Publishing/index.tsx (hunk 2 above)
Comment (kanaabe) [reply src/Components/Publishing/index.tsx]: Ah yeah the latter is definitely what we want. Good call!

**Pull request merged**